app/src/main/res/values/strings.xml

```diff
@@ -370,7 +370,7 @@
 
   <string name="write_storage_permission_rationale_for_image_share">We need your permission to access the external storage of your device in order to upload images.</string>
   <string name="nearby_notification_dismiss_message">You won\'t see the nearest place that needs pictures anymore. However, you can re-enable this notification in Settings if you wish.</string>
-  <string name="step_count">Step %1$d of %2$d: %3$s</string>
+  <string name="step_count">%1$d/%2$d: %3$s</string>
```

Comment:
Hi @vivek-gite , thanks for the changes but I think it looks irrelevant. With this change you change the string for all of the languages. What should be done is making the field ıi element visible without changing the text.